components/home-intro.tsx

```diff
@@ -43,6 +43,11 @@ export default function HomeIntro ({ isFlat = false }) {
         Custom badge scale
         <a href='/badge/docker/v1.2.3/blue?icon=docker&scale=2'>e.g.</a>
       </li>
+      <li>
+        <code>cache</code>
+        Live badge are cached in cdn for 24hrs (86400), you may limit it to 5min (300) at least.
```

Comment:
"you may limit it to a minimum of 5min (300)." would be a better wording imo

Reply:
Great! Updated in fca1d94